swift/BUILD.bazel

```diff
@@ -64,6 +64,13 @@ pkg_runfiles(
     prefix = "tools/" + codeql_platform,
 )
 
+pkg_filegroup(
+    name = "resource-dir-arch",
+    srcs = ["//swift/third_party/swift-llvm-support:swift-resource-dir"],
+    prefix = "resource-dir/" + codeql_platform,
+    visibility = ["//visibility:public"],
+)
+
 pkg_files(
     name = "swift-test-sdk-arch",
     srcs = ["//swift/third_party/swift-llvm-support:swift-test-sdk"],
```

Comment:
Is the test-sdk still required? can't the resource-dir replace the need for the separate sdk? In case the answer is yes, this can probably wait the follow-up PRs

Reply:
I'm not 100% sure, so decided to keep it as is for now, Happy to look into it separately.